exercises/08-execute-javascript.md

```diff
@@ -136,7 +172,7 @@ so we can just simply use that as the locator.
 
 Browser library has builtin waiting and no separate validation keywords. Instead, you can use `Get Text`
 to validate the text automatically. Python validations such as `==`, `!=`, `contains`, and `not contains`
-are available for validation. The syntax is Pythonesque:
+are available for validation. The syntax is Python-esque:
```

Comment:
Since Browser library evaluations should be explained earlier (see previous comment), this paragraph should be moved there.